multigeiger/userdefines-example.h

```diff
@@ -56,3 +56,10 @@
 // Note: The TTN configuration needs to be done in lorawan.cpp (starting at line 65).
 #define SEND2LORA false
 
+// Send data via BLE?
+// Device provides "Heart Rate Service" (0x180D) and these characteristics
```

Comment:
Hmm, is there nothing rather generic we could use so we must abuse a `Heart Rate` measuring mechanism?

Reply:
OTOH, if the CPM goes rather high, your heart rate might also go up. :-X

Reply:
The intention was that this way the Multigeiger can be connected to different devices to display the CPM - all that can display Heart Rate via BLE. But typically not both CPM and real Heart Rate, though :)

Reply:
ah, ok.